lib/src/slices/data_bkg/data_bkg_sv_email_prov.dart

```diff
@@ -0,0 +1,20 @@
+import '../api_email_msg/model/api_email_msg_model.dart';
+import 'model/data_bkg_model_page.dart';
+
+abstract class DataBkgSvEmailProvInterface {
```

Comment:
Class names should always match file names. This class should be named DataBkgServiceEmailInterface